Fixes build failure against random 1.0.1+

[28 of 36] Compiling Bio.Util.TestBase ( Bio/Util/TestBase.hs, dist/build/qc/qc-tmp/Bio/Util/TestBase.o )

Bio/Util/TestBase.hs:75:10:
    Duplicate instance declarations:
      instance Random Word8 -- Defined at Bio/Util/TestBase.hs:75:10-21
      instance Random Word8 -- Defined in System.Random
diff --git a/Bio/Util/TestBase.hs b/Bio/Util/TestBase.hs
index d509af7..75dab65 100644
--- a/Bio/Util/TestBase.hs
+++ b/Bio/Util/TestBase.hs
@@ -1,4 +1,4 @@
-{-# Language ExistentialQuantification #-}
+{-# Language ExistentialQuantification, CPP #-}
 
 module Bio.Util.TestBase where
 
@@ -71,10 +71,11 @@ genPositiveOffset = do scale <- chooseInteger (1, 13)
     where chooseInteger :: (Integer, Integer) -> Gen Integer
           chooseInteger = choose
 
-
+#if !MIN_VERSION_random(1,0,1)
 instance Random Word8 where
     randomR = integralRandomR
     random = randomR (minBound,maxBound)
+#endif
 
 instance Arbitrary Nucleotide where
     arbitrary = elements (map N "aaacccgggtttn")
